summaries/2020/10086.md

```diff
@@ -2,6 +2,25 @@
 
 https://developer.apple.com/wwdc20/10086
 
-Presenters: _Example Guy, Another Person_
+Presenter: _Mark Mikin_
 
-##### TO-DO! You can contribute to this session, please see [CONTRIBUTING.md](CONTRIBUTING.md)
+- This is one of those "why" sessions. Why we should build something?
+- **Intelligent System Experience**
+  - The main idea is to be proactive
+  - Intelligence is how the OS works with the apps that people use every day to make the "every day" easier for people
+  - Intelligence is design (it should be viewed as a design practice)
+- **Living design**. What is that?
+  - The core job of the designer is to help people accomplish something
+  - One of the key ways a designer can enable this is by leveraging the familiar
```

Comment:
```suggestion
  - One of the key ways a designer can leverage this is by using familiar elements
```

Reply:
Or:

```suggestion
  - One of the key ways a designer can leverage this is by using elements familiar to the user
```

Reply:
Second one sounds great for me. Thank you!